symbolic-common/src/types.rs

```diff
@@ -702,10 +730,36 @@ impl<'a> Name<'a> {
         Name {
             string: string.into(),
             lang: Language::Unknown,
+            mangling: Default::default(),
```

Comment:
Let's be explicit in all places where we use `NameMangling::Unknown`, since this now has significant semantics.

```suggestion
            mangling: NameMangling::Unknown,
```